media/css/mozorg/home/home-2018.scss

```diff
@@ -5,15 +5,13 @@
 $font-path: '/media/fonts';
 $image-path: '/media/protocol/img';
 
-@import "../../../protocol/css/includes/lib";
-@import "../../../protocol/css/components/billboard";
-@import "../../../protocol/css/components/button";
-@import "../../../protocol/css/components/download-button";
```

Comment:
I'm removing the `download-button` component here, as it doesn't seem to be used but please check (the page is using custom styles).